src/odemis/gui/comp/overlay/cryo_feature.py

```diff
@@ -128,9 +140,11 @@ def on_dbl_click(self, evt):
             v_pos = evt.Position
```

Comment:
In cryo_feature.py, self.active is incorrectly used. Instead of self.active, it should be self.active.value as it is a VA, Maybe it is a good time to correct it?

Reply:
good catch, updated

Reply:
@nandishjpatel found that error with self.active. Coming back to the question, The thing is self.active.value is always set to True, it is not made False anywhere. It will work anyways. 

Additionally, the event of on_motion needs to be handled properly if self.active.value is used.  The last line should be WorldOverlay.on_motion(self, evt) for the event to end properly.

Reply:
I'm not sure I understand what self.active does?  I thought it was for when the tool was selected?

Reply:
I think self.active.value is used such that correct overlay class is used specially when there are multiple overlay classes. All the events functions like double click, left up, left down and on motion will work only if self.active.value is true. When the tool is selected, based on the event it decides either to add a new feature or modify an existing feature. 

As it is always true from the beginning when the overlay is initialized/added to the canvas, all the event functions will be active and work as before (i.e. when self.active is used)

As a precaution, I talked about on_motion event because this event does not end properly because evt.Skip() is not the last line of execution. This is  just in case when self.active.value is set to False in future changes.